transitions/registration.js

```diff
@@ -333,14 +340,43 @@ module.exports = {
         var doc = options.doc,
             db = db || options.db;
 
-        transitionUtils.addUniqueId(db, doc, callback);
+        if (options.params.patient_id) {
+            var located = doc.fields[options.params.patient_id];
```

Comment:
Sorry yeah, that was a left-over from a different strategy, you're right it makes no sense now.